package.json

```diff
@@ -1,20 +1,24 @@
 {
-    "private": true,
-    "scripts": {
-        "ci": "yarn lint",
-        "commit": "git-cz",
-        "lint": "lerna run lint",
-        "postinstall": "lerna run build"
-    },
-    "workspaces": [
-        "packages/*"
-    ],
-    "devDependencies": {
-        "commitizen": "2.9.6",
-        "lerna": "2.8.0"
-    },
-    "engines": {
-        "node": ">=8.9.0",
-        "yarn": "1.3.2"
-    }
+  "private": true,
+  "scripts": {
+    "ci": "yarn lint",
+    "commit": "git-cz",
+    "deps:fix": "npx @times-components/depend -l . -e '{node_modules/@times-components/*,.}/package.json' -s majority --fix",
+    "deps:lint": "npx @times-components/depend -l . -e '{node_modules/@times-components/*,.}/package.json' -s majority --hint",
+    "deps:list": "npx @times-components/depend -l . -e '{node_modules/@times-components/*,.}/package.json' -s majority --list",
+    "lint": "lerna run lint",
```

Comment:
this now should be `yarn run deps:lint && lerna run lint`

Reply:
Added to prelint 27a39b0